post/clang_tidy_review/clang_tidy_review/__init__.py

```diff
@@ -85,12 +94,11 @@ def build_clang_tidy_warnings(
     print(f"Took: {end - start}")
 
 
-def clang_tidy_version(clang_tidy_binary: str):
+def clang_tidy_version(clang_tidy_binary: pathlib.Path):
     try:
         version_out = subprocess.run(
-            f"{clang_tidy_binary} --version",
+            [clang_tidy_binary, "--version"],
```

Comment:
This change in quoting is also ok because of using `subprocess.run` with `shell=False`?

Reply:
Exactly, there's no confusion about slashes when running with `shell=False` while as a string, there was.
As an added safety, I also made sure `clang_tidy_binary` is a `pathlib.Path` which gives the correct slashes.